test/integration/clusterresourceoverride_admission_test.go

```diff
@@ -153,6 +153,9 @@ func setupClusterResourceOverrideTest(t *testing.T, pluginConfig *overrideapi.Cl
 	if err := testutil.WaitForClusterResourceQuotaCRDAvailable(clusterAdminClientConfig); err != nil {
 		t.Fatal(err)
 	}
+	if err := testutil.WaitForSecurityContextConstraintCRDAvailable(clusterAdminClientConfig); err != nil {
```

Comment:
Put this into `test/integration/runner_test.go::executeTests()`, SCCs are important for being able to run pretty much anything.

Reply:
done (you did it) : )